src/pkgcheck/checks/python.py

```diff
@@ -52,7 +52,7 @@ def is_python_interpreter(pkg):
         # ignore python:2.7 deps since they are being phased out from eclass
         # support
         return pkg.slot is None or not pkg.slot.startswith("2")
-    return pkg.key in ("dev-python/pypy3",)
+    return pkg.key in ("dev-python/pypy3", "dev-lang/python-freethreading")
```

Comment:
Don't we need to update this to pypy3_10 and so on?

Reply:
No, we depend on `dev-python/pypy3:=`.

Reply:
Thanks. A comment may help but not required.